services/audit/pkg/types/types.go

```diff
@@ -205,6 +205,31 @@ type AuditEventSpaceDeleted struct {
 	AuditEventSpaces
 }
 
+// AuditEventSpaceShared is the event logged when a space is shared
+type AuditEventSpaceShared struct {
+	AuditEventSpaces
+
+	GranteeUserID  string
+	GranteeGroupID string
+}
+
+// AuditEventSpaceUnshared is the event logged when a space is unshared
+type AuditEventSpaceUnshared struct {
+	AuditEventSpaces
+
+	GranteeUserID  string
+	GranteeGroupID string
+}
+
+// AuditEventSpaceUpdated is the event logged when a space is updated
+type AuditEventSpaceUpdated struct {
+	AuditEventSpaces
+
+	Name          string
```

Comment:
Can we also add the updated description?

Reply:
Checked it. It is inside the opaque map

Reply:
:+1: